### DISCUSSION TITLE
Pass user-data cloud-init file for VPS servers on reinstall / setup

### DISCUSSION THREAD

**Discussion by BitFis** [ovh/manager#8406] [Ideas]
Hello

I couldn't find a way to inject a custom cloud-init user-data file to help with initialisation of my VPS servers. Would be very handy if I could upload my custom cloud-init config file on creation or recreation / reinstall of a VPS either via API or/and UI.

**Comment by dubaiclean** [comment]
I hope so you will find a better answer to your problem. I have also a problem regarding a good content writer for my site [Dubai cleaning services](https://dubaiclean.com/). if you know about it then let me know
